### PR TITLE
chore: remove all walrus operator usage

### PR DESCRIPTION
**Pull request by NiceAesth**
walrus

### CODE REVIEW

File: app/api/domains/api.py [hunk 1]
```diff
@@ -760,7 +760,8 @@ async def api_get_match(
     """Return information of a given multiplayer match."""
     # TODO: eventually, this should contain recent score info.
 
-    if not (match := app.state.sessions.matches[match_id]):
+    match = app.state.sessions.matches[match_id]
```
Comment (cmyui) [inline app/api/domains/api.py:763]: lmao this syntax is so unintuitive, what was i thinking

Comment (NiceAesth) [reply app/api/domains/api.py:763]: oh right, i should change that to get()

Comment (NiceAesth) [reply app/api/domains/api.py]: to be fair i think `api_get_match` should be refactored because it looks like a dumpster fire rn

**Comment by cmyui** [comment]
u making me feel bad with this image

File: app/api/domains/api.py (hunk 1 above)
Comment (cmyui) [reply app/api/domains/api.py]: yeah i'm actually going to revert the commit you added - there are multiple additional uses of this and we can leave this for another pr

Comment (NiceAesth) [reply app/api/domains/api.py:763]: okok

File: app/commands.py [hunk 2]
```diff
@@ -130,11 +130,15 @@ def wrapper(f: Callback) -> Callback:
 
 # not sure if this should be in glob or not,
 # trying to think of some use cases lol...
```
Comment (cmyui) [inline app/commands.py:132]: comments lying again
```suggestion
```

File: app/commands.py [hunk 3]
```diff
@@ -1615,17 +1639,18 @@ async def mp_addref(ctx: Context, match: Match) -> Optional[str]:
...
 
-    if t in match.refs:
-        return f"{t} is already a match referee!"
+    if target in match.refs:
+        return f"{target} is already a match referee!"
 
-    match._refs.add(t)
-    return f"{t.name} added to match referees."
+    match._refs.add(target)
+    return f"{target.name} added to match referees."
```
Comment (cmyui) [inline app/commands.py:1651]: lol wtf i'm using target in one return, and target.name in another

File: app/commands.py [hunk 4]
```diff
@@ -1635,17 +1660,18 @@ async def mp_rmref(ctx: Context, match: Match) -> Optional[str]:
...
+        return f"{target} is not a match referee!"
 
-    if t is match.host:
+    if target is match.host:
         return "The host is always a referee!"
 
-    match._refs.remove(t)
-    return f"{t.name} removed from match referees."
+    match._refs.remove(target)
+    return f"{target.name} removed from match referees."
```
Comment (cmyui) [inline app/commands.py:1672]: seems this is a repetitive thing.. maybe best to keep for another pr

Comment (NiceAesth) [reply app/commands.py:1672]: `target.name` doesn't seem to be used that many times

File: app/settings.py [hunk 5]
```diff
@@ -20,7 +20,7 @@ def read_list(value: str) -> list[str]:
 
 
 SERVER_ADDR = os.environ["SERVER_ADDR"]
-SERVER_PORT = int(v) if (v := os.getenv("SERVER_PORT", None)) else None
+SERVER_PORT = int(os.getenv("SERVER_PORT", 0)) or None
```
Comment (cmyui) [inline app/settings.py:23]: my brain

Comment (minisbett) [reply app/settings.py:23]: bro your fucking server is dying

**Review by cmyui — APPROVED** [review]
+1 👍🏻
